contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -9894,3 +9879,12 @@ CheckDeclareAtAtGlobalVarName(const std::string name, int lineNr)
 		throw PGErrorWrapperException(ERROR, ERRCODE_SYNTAX_ERROR, format_errmsg("Incorrect syntax near '%s'.", name.c_str()), lineNr, 0);
 	}
 }
+
+static antlr4::tree::TerminalNode *
+getTokenFromFunctionOption(TSqlParser::Function_optionContext* o) {
+    if (o->execute_as_clause())
+        return o->execute_as_clause()->CALLER();
+    if (o->inline_clause())
+        return o->inline_clause()->INLINE();
+    return o->SCHEMABINDING();
```

Comment:
SCHEMABINDING is optional, right? then we should check `if (o->schemabinding())` then only return something. It will be neat code in that way